spec/overview.html

```diff
@@ -73,6 +73,14 @@ <h1>Implementation Dependencies</h1>
       </li>
     </ul>
 
+    <emu-note>
+      Changes to the set of locales, currencies, calendars, numbering systems, and other enumerable items made available by ECMAScript hosts must not result in two users using the same version of the same host on the same platform becoming distinguishable from each other. This constraint is imposed to reduce the fingerprinting risk inherent in internationalization, and may be relaxed in future revisions.
```

Comment:
Should this be normative? The line below should be informative, but it sounds like this is expected to be normative?

Reply:
This should indeed be normative, though it's in that fuzzy space where it's a normative change that doesn't impact any extant implementation.